src/plugins/unified_search/public/services.ts

```diff
@@ -11,8 +11,9 @@ import { createGetterSetter } from '@kbn/kibana-utils-plugin/public';
 import { DataViewsContract } from '@kbn/data-views-plugin/public';
 import { AutocompleteStart } from '.';
 
-export const [getIndexPatterns, setIndexPatterns] =
-  createGetterSetter<DataViewsContract>('IndexPatterns');
+export const [getDataViews, setDataViews] = createGetterSetter<DataViewsContract>('IndexPatterns');
+
+export const [getIndexPatterns, setIndexPatterns] = [getDataViews, setDataViews];
```

Comment:
why we need it?  you can just rename the methods

Reply:
If I rename this then other plugins that import this method will have to change (rename this method). I initially sought to make changes only in unified search.

Reply:
other plugins should not normally call these methods! In some cases, this may come from tests, but this is not quite the right approach.